subprojects/docs/src/samples/build-organization/standalone-precompiled-plugins/groovy/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'groovy-gradle-plugin'
+    id 'maven-publish'
+}
+
+// tag::repositories-and-dependencies[]
+repositories {
+    jcenter()
```

Comment:
❓ I believe jcenter isn't necessary given all used dependencies should be found in the plugin portal. Same in the kotlin sample.

Reply:
True, works without jcenter. I thought I needed it for junit/spock dependencies.